pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -60,6 +68,17 @@ type NfdTopologyUpdater interface {
 type staticNodeInfo struct {
 	nodeName string
 	tmPolicy string
+	tmScope  string
+}
+
+func newStaticNodeInfo(policy, scope string) *staticNodeInfo {
```

Comment:
nit/nonblocking: I'd just return a struct (not a pointer to a struct) and trust the compiler.
In general, returning pointers *may* allow faster copying of objects (but it should actually be measured) but also contributes to the GC pressure (which, OTOH, should also be measured).
What I'm trying to say is that returning pointers is not necessarily the faster option,

Reply:
note: I'd NOT resubmit just for this.

Reply:
I want to add unit tests, so I'll change it when I submit them.